<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hierarchy</class>
 <widget class="QWidget" name="Hierarchy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_AddEntity">
     <property name="text">
      <string>Add Entity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_RemoveEntity">
     <property name="text">
      <string>Remove Entity</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
